scout/adapter/mongo/hgnc.py

```diff
@@ -43,6 +43,43 @@ def load_hgnc_bulk(self, gene_objs):
 
         return result
 
+    def hgnc_gene_caption(self, hgnc_identifier, build="37"):
+        """Fetch the current hgnc gene symbol and similar caption info for a gene in a lightweight dict
+        Avoid populating transcripts, exons etc that would be added on a full gene object. Use hgnc_gene() if
+        you need to use those.
+
+        Args:
+            hgnc_identifier(int)
+            build(str)
```

Comment:
What about defaulting to None and including it in the query only if it has a value in ["37","38"]? 
Because I'm looking at your proposed change in the function /scout/server/blueprints/institutes/views.gene_variants and there you don't pass any build (because I guess it could be both builds). But it won't work for build 38 cases because it will default to build 37.

You could perhaps pass nothing and not use build in the query? It will be only one result anyway because you are using find_one.

Reply:
Yep! It was the same before, and honestly mostly ppl use symbols rather than HGNC-ids in the search field, but you are right. Fixed below by asking for captions from 38 if none was found for digit in 37.